.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+### Description
+Please explain the changes you made here.
+
+### Checklist
+- [ ] Created tests which fail without the change (if possible)
+- [ ] All tests passing
```

Comment:
Suggestion (non-blocking): We've got a check that blocks the PR if tests are failing so might not need this.

Reply:
Good point! Removed